### PR TITLE
[DEVELOPER-3405] Connectors assembly type

### LINKED ISSUE #3405: (3) List view assembly for Connectors content type
**Estimate:** 3

Content at this page:

https://developers.redhat.com/products/fuse/connectors

pulls from the Connectors content type in Drupal.

This page (and the whole Fuse product page triptych) is being converted to the new product page design.

What we need is a net-new assembly that uses the existing List View design, displaying the value in the `Title` field as a hyperlink to the URL in the `Connector Details URL` field, followed by the text in the `Connector Short Description` field as the link description.

Screen Shot 2019-11-25 at 10 55 32 AM

### PR DESCRIPTION
**Pull request by jordanpagewhite**
### Issue Link

Resolves #3405 

### Stakeholders

@joeschram 
@sjcox-rh 
@gdoyle1 

### Verification Process

- Go here: Any new, assembly-based product page ex. /products/openjdk/overview
- In a separate tab, open the edit page: /node/207365/edit
- In the Overview Page Content field, create a Connectors assembly.
  - This should be available from the Overview Page Content, Download Page Content and Getting Started page content fields.
- Save the node
- Refresh the Product overview page in your other tab

You should see this:

Screen Shot 2019-12-05 at 1 50 15 PM

### CODE REVIEW

**Comment by jordanpagewhite** [comment]
rebuild this please

**Comment by alistairmc** [comment]
rebuild this please

**Comment by jordanpagewhite** [comment]
rebuild this please

**Comment by jordanpagewhite** [comment]
rebuild this please

File: _docker/drupal/drupal-filesystem/web/config/sync/core.entity_form_display.assembly.connectors.default.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+id: assembly.connectors.default
+targetEntityType: assembly
+bundle: connectors
+mode: default
+content:
+  field_audience_selection:
+    weight: 100
+    settings: {  }
+    third_party_settings: {  }
+    type: options_select
```
Comment (rogeralloymagnetic) [inline _docker/drupal/drupal-filesystem/web/config/sync/core.entity_form_display.assembly.connectors.default.yml]: This should be checkboxes to match the rest of the site.

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/templates/node/node--connectors--list-view.html.twig [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{% set classes = [
+  'pf-l-grid',
+  'pf-m-gutter',
+  'product-version'
+]%}
+<div{{ attributes.addClass(classes) }}{{ audience_selection }}>
+  <div class="pf-l-grid__item product-version__file-label-wrapper">
+    <p class="product-version__file-label">{{ label }}</p>
```
Comment (rogeralloymagnetic) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/templates/node/node--connectors--list-view.html.twig]: The original ticket states: "...displaying the value in the Title field as a hyperlink to the URL in the Connector Details URL field..."
It seems like this should link out.

Comment (jordanpagewhite) [reply _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/templates/node/node--connectors--list-view.html.twig]: Ah, you are right. Thank you!

File: _docker/drupal/drupal-filesystem/web/config/sync/core.entity_form_display.assembly.connectors.default.yml (hunk 1 above)
Comment (jordanpagewhite) [reply _docker/drupal/drupal-filesystem/web/config/sync/core.entity_form_display.assembly.connectors.default.yml]: I'll take care of that. Thanks

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/templates/node/node--connectors--list-view.html.twig (hunk 2 above)
Comment (rogeralloymagnetic) [reply _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/templates/node/node--connectors--list-view.html.twig]: This looks good.

File: _docker/drupal/drupal-filesystem/web/config/sync/core.entity_form_display.assembly.connectors.default.yml (hunk 1 above)
Comment (rogeralloymagnetic) [reply _docker/drupal/drupal-filesystem/web/config/sync/core.entity_form_display.assembly.connectors.default.yml]: This just needs to get moved up in order to be directly underneath the `name` field.

Comment (jordanpagewhite) [reply _docker/drupal/drupal-filesystem/web/config/sync/core.entity_form_display.assembly.connectors.default.yml]: Good point. Thank you! I just pushed up that change.

Comment (rogeralloymagnetic) [reply _docker/drupal/drupal-filesystem/web/config/sync/core.entity_form_display.assembly.connectors.default.yml]: looks good

**Comment by robpblake** [comment]
rebuild this please

**Pull request merged**